src/EPR.SubsidiaryBulkUpload.Application/Models/BulkUpdateErrors.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace EPR.SubsidiaryBulkUpload.Application.Models;
+
+public static class BulkUpdateErrors
+{
+    public const int FileEmptyError = 100;
+
+    public const int InvalidHeader = 101;
+
+    public const int OrganisationIdRequired = 102;
+    public const string OrganisationIdRequiredMessage = "organisation_id is required.";
+
+    public const int OrganisationNameRequired = 103;
+    public const string OrganisationNameRequiredMessage = "organisation_name is required.";
+
+    public const int CompaniewsHouseNumberRequired = 104;
```

Comment:
Typo on both of these lines - Companiews

Reply:
fixed